_posts/2019-02-24-The-Intelligent-Investor.md

```diff
@@ -0,0 +1,152 @@
+
+### When is it a Good Point to Become an Investor?
+
+Everyone can be a mix, but taking big risks when you're in debt or living hand to mouth is flirting with disaster. The money used for  investing is money set aside for this purpose after all your other needs have been met and you're safe.
+
+Benjamin Graham doesn't go into a ton of detail of this, but I think it is safe to say that these are reasonable criteria starts to a list, meaning more reasons exist but these were the ones I could think of on a Sunday afternoon:
+
+  1) You have to have **no debt at all**, when I say that I mean: car payments, student loan debt, credit cards, a large mortgage you need to pay off monthly. Debt can be defined as **owing any entity money**.
+
+  2) Make sure you have a stable income source with excess money to use: jobs, side hustles, social security, winnings from lay suites, etc.
+
+  3) Do you have a safety net of between 6 - 12 months of living expenses saved up? When the shit goes down you better be ready.
+
+
+### Why Am I Focusing on Defensive Investors and not talking about Enterprising Investors?
+
+Many people try, and typically fail, to beat the market averages for many reasons, but the reality is that most people think they're above average<sup>3</sup> when in fact that's statistically impossible. Can _YOU_ beat the market consistently for decades? You can try, but it is likely that you will fail due to sheer time required, temperamental and intellectual advantage within the market, and the element of markets becoming more efficient particularly at the large capital end i.e. companies in the S&P 500.
+
+To truly beat the market over long time spans one has to have patience, extreme patience, and an ability to focus on the larger goal while keeping their emotions in check for decades. It is a slow, turbulent, and boring process for most. People like Warren Buffett, Joel Greenblatt, Charlie Munger, and Ray Dalio are not just rare, they're exceptional in ways beyond just mental powers that we can measure, like IQ.
+
+Note: I included a link at the bottom to the subject of Dysrationalia, it is a fascinating topic and one that has no received enough attention in the psychology community.
+
+If you read Vanguard's [The Case for Index Funds](https://pressroom.vanguard.com/nonindexed/Updated_The_Case_for_Index_Fund_Investing_4.9.2014.pdf) you will realize most "experts" are nothing more than highly paid money managers who on the whole detract from your portfolio returns by "managing" them when a simple index fund would beat 80%+ of these managers. This is the evil twin of compounded return, compounded loss. But wait you say, what about the other ~20+% of managers, why don't I just go with them? That group is not a consistent batch of people so how are you supposed to pick the best fund managers if they don't even know who they are?
```

Comment:
`This is the evil twin of compounded return, compounded loss.`
Great sentence.

Reply:
**"** But wait **",** you say, **"** what about the other ~20+% of managers, why don't I just go with them? **"**.